x-pack/plugins/index_management/__jest__/components/index_table.test.js

```diff
@@ -254,15 +261,27 @@ describe('index table', () => {
     expect(button.text()).toEqual('Manage 2 indices');
   });
 
-  test('should show system indices only when the switch is turned on', async () => {
+  test('should show hidden indices only when the switch is turned on', async () => {
     const rendered = mountWithIntl(component);
     await runAllPromises();
     rendered.update();
 
-    snapshot(rendered.find('.euiPagination li').map((item) => item.text()));
-    const switchControl = rendered.find('.euiSwitch__button');
+    // We have manually defined above that the `.admin1` and `.admin3` indices are hidden.
+    // We **don't** expect them to be in this list as by default we don't show hidden indices
+    let indicesInTable = namesText(rendered);
+    expect(indicesInTable).not.toContain('.admin1');
+    expect(indicesInTable).not.toContain('.admin3');
```

Comment:
This test makes sense for 8.0, but does it make sense for 7.16? In 7.16 wouldn't we want to assert that all dot-prefixed indices are hidden here, and revealed on lines 282-283?

Reply:
That's what the snapshots should give us. But you are right it is not very explicit (that's why I never use snapshots). I added a test to make it explicit (https://github.com/elastic/kibana/pull/112990/commits/dc292d5181cce52fa1e0ad2e92e7492c5afbed12)